ratpack/src/main/groovy/com/baeldung/Ratpack.groovy

```diff
@@ -0,0 +1,61 @@
+package com.baeldung;
+
+@Grab('io.ratpack:ratpack-groovy:1.6.1')
+import static ratpack.groovy.Groovy.ratpack
+
+import com.baeldung.model.User
+import com.google.common.reflect.TypeToken
+import ratpack.exec.Promise
+import ratpack.handling.Context
+import ratpack.jackson.Jackson
+import groovy.sql.Sql
+import ratpack.hikari.HikariModule
+
+ratpack {
+    bindings {
+        module(HikariModule) { config ->
+            config.dataSourceClassName = 'org.h2.jdbcx.JdbcDataSource'
+            config.addDataSourceProperty('URL', "jdbc:h2:mem:devDB;INIT=RUNSCRIPT FROM 'classpath:/User.sql'")
+        }
+    }
+
+
+    handlers {
+        
+        get {
+            render 'Hello World from Ratpck with Groovy!!'
+        }
+        
+        get("greet/:name"){Context ctx ->
+            render "Hello " + ctx.getPathTokens().get("name") + " !!!"
```

Comment:
(being picky, I'd not have the space before `!!!`

Reply:
Would it be cleaner to use the inferred single parameter here? or is it better to declare `Context`?

Reply:
ctx is the parameter available in the handlers.
It is better to declare the Context as reader will get to know the parameter available inside closure if of Context type and hence can understand all the available properties and behavior of it.